### PR TITLE
Convert docs components to native classes

### PR DESCRIPTION
**Pull request by SanderKnauff**
This resolves a number of linting warnings in the docs app.
It is primarily focused on fixing the `ember/no-classic-classes` lint rule.

There is also a small fix for the rendering of the markdown changelogs, due to `ember-cli-showdown` 8.0.0 removing the ability to use positional parameters.

### CODE REVIEW

**Comment by SanderKnauff** [comment]
I see that my merge from master into my working branch caused some chaos. I'm going to rebuild the branch to clean it up a bit.

File: docs/app/models/login.js [hunk 1]
```diff
@@ -18,9 +19,8 @@ const Validations = buildValidations({
   email: [validator('presence', true), validator('format', { type: 'email' })],
 });
 
-export default EmberObject.extend(Validations, {
-  email: null,
-  password: null,
-  rememberMe: false,
```
Comment (jelhan) [inline docs/app/models/login.js:24]: Good catch that `rememberMe` wasn't actually used in the template. :clap:

File: docs/app/templates/application.hbs [hunk 2]
```diff
@@ -1,85 +1,78 @@
-{{#unless this.hideNav}}
```
Comment (jelhan) [inline docs/app/templates/application.hbs:1]: Good catch that `hideNav` could never be `true` as there isn't any route containing `'acceptance'`. :clap:

**Review by jelhan — APPROVED** [review]
Looks good to me!

Thanks a lot for addressing this long outstanding technical debt!

**Pull request merged**